doc/tutorials/running_deb_as_serv_vm.rst

```diff
@@ -44,88 +44,93 @@ Validated Versions
 ******************
 
 - **Debian version:** 10.0 (buster)
-- **ACRN hypervisor tag:** acrn-2019w35.1-140000p
-- **Debian Service VM Kernel version:** 4.19.68-84.iot-lts2018-sos
+- **ACRN hypervisor tag:** acrn-2020w40.1-180000p
+- **Debian Service VM Kernel version:** 5.4.43-PKT-200203T060100Z
```

Comment:
Further below, we instruct the use to clone and check out `release_2.2` from https://github.com/projectacrn/acrn-kernel, but that version does not match the tag you indicate here. Can you correct this by fixing the tag and/or the instructions to match what we have tested and validated?

Reply:
update with tag: ** acrn-2020w40.1-180000p